packages/@okta/courage-for-signin-widget/README.md

```diff
@@ -0,0 +1,11 @@
+# @okta/courage-for-signin-widget
+
+Generates a custom version of Courage for the Sign-In Widget. 
+It's intend for internal developers and please open a ticket if any questions.
+
+## Steps
+
+0. make sure VPN is connected
+1. upgrade courage version. e.g. `yarn upgrade @okta/courage@4.4.0-beta.xxxxxx`
+2. `yarn build`
+3. check file `../okta-for-signin-widget.js` which may be changed depending on the change set since last upgrade
```

Comment:
A couple of things in this line
- name of the file should be updated to `courage-for-signin-widget.js`
- we can re-word this to a little more explicit instructions like
```
commit file ..okta/courage-for-signin-widget.js if it has changes. This file will have changes only if the relevant components of courage that we consume in the Sign-In Widget have changed.
```